Philips.CodeAnalysis.Test/Common/AllowedSymbolsTest.cs

```diff
@@ -96,7 +96,8 @@ private void AnalyzeMethod(SymbolAnalysisContext context)
 		public void AllowedSymbolShouldBeReportDiagnostics(string nsName, string typeName, string methodName)
 		{
 			var file = GenerateCodeFile(nsName, typeName, methodName);
-			VerifyDiagnostic(file);
+			DiagnosticResult diagnosticResult = GetDiagnosticResult();
```

Comment:
Use var here, it's clear what the type is

Reply:
done